simple-backend/nlpviewer_backend/urls.py

```diff
@@ -70,8 +70,8 @@
     path('ontology_from_doc/<int:document_id>', document.get_doc_ontology_pack),
     path('config_from_doc/<int:document_id>', document.get_doc_project_config),
 
-    path('nlp/load/<model_name>', nlp.load_model),
-    path('nlp/<int:document_id>/<model_name>', nlp.run_pipeline),
+    path('nlp/load/<int:document_id>', nlp.load_model),
+    path('nlp/<int:document_id>', nlp.run_pipeline),
```

Comment:
Similar to above, using `document_id` here to load model or run pipeline is confusing. We should make the logic clear here.

Reply:
I think maybe for the `run_pipeline` function, `document_id` is required because it can specify the input `DataPack` to the pipeline.

Reply:
I see, then that one can be kept.